### PR TITLE
like-dislike and bookmark functionality

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
Hello, i have Added GET ALL Bookmarks, like post , Dislike post, bookmark post, remove from bookmarkspost functionality, in social media app. kindly review it and let me know what do think about it !
PR: https://github.com/kgajriaAgarwal/blithe-in/pull/8
 src/Helpers/Services/actions.jsx
src/app/slice/postSlice.js
src/pages/Shared/Feed/Feed.js
 src/pages/Shared/FeedCard/FeedCard.js
 src/pages/Shared/NavBar/NavBar.js
 src/pages/Bookmark/
Preview: https://blithe-in.netlify.app/

### CODE REVIEW

File: src/pages/Shared/NavBar/NavBar.js [hunk 1]
```diff
@@ -93,24 +95,34 @@ export const NavBar = (props) =>  {
...
                         // value={value}
                         //onChange={handleChange}
                         variant="fullWidth"
                         textColor="secondary"
                         color='white'
+                        // value={
+                        //     history.location.pathname !== "/"
+                        //       ? history.location.pathname
+                        //       : false
+                        //   }
```
Comment (amantulsyan35) [inline src/pages/Shared/NavBar/NavBar.js:110]: can remove comments

File: src/Helpers/Services/actions.jsx [hunk 2]
```diff
@@ -202,4 +202,65 @@ export const actionDownvoteComment = async (data) => {
...
+export const actionLikePost = async (data) => {
+  const response = await mainApiService("actionLikePost", data);
+  if (response.data &&(response.status === 201 || response.status === 200)) {
+    return response;
+    }else{
+      console.log("response.data.errors:", response.data.errors);
+      // showErrorToast(response.data.errors[0])
+      showErrorToast("Unexpected error")
+  }
+};
```
Comment (amantulsyan35) [inline src/Helpers/Services/actions.jsx:217]: can be error handled with try catch blocks

Comment (kgajriaAgarwal) [reply src/Helpers/Services/actions.jsx:217]: Done that is  postslice.js

File: src/Helpers/Services/actions.jsx [hunk 3]
```diff
@@ -202,4 +202,65 @@ export const actionDownvoteComment = async (data) => {
...
+
+//actionBookmarkPost
+export const actionBookmarkPost = async (data) => {
+  const response = await mainApiService("actionBookmarkPost", data);
+  if (response.data &&(response.status === 201 || response.status === 200)) {
```
Comment (MayankKumar10) [inline src/Helpers/Services/actions.jsx:245]: Suggestion: you can store res.status into a variable used that value instead so you have to write less code

File: src/pages/Shared/NavBar/NavBar.js [hunk 4]
```diff
@@ -129,26 +141,30 @@ export const NavBar = (props) =>  {
...
                         </Badge>} 
-                        value="/jobs" 
+                        // value="/users/bookmark" 
+                        component={Link}
+                        to={"/users/bookmark"}
                         />
 
-                        <CustomTab icon={<Badge badgeContent={4} color="error" >
+                        {/* <CustomTab icon={<Badge badgeContent={4} color="error" >
                             <Tooltip title="Notifications">
```
Comment (MayankKumar10) [inline src/pages/Shared/NavBar/NavBar.js:161]: Suggestion: remove comments

**Review by MayankKumar10 — APPROVED** [review]
Look into these suggestions everything else looks good to me you can merge it.

File: src/Helpers/Services/actions.jsx [hunk 5]
```diff
@@ -202,4 +202,65 @@ export const actionDownvoteComment = async (data) => {
...
+};
+
+export const actionDislikePost = async (data) => {
+  const response = await mainApiService("actionDislikePost", data);
+  if (response.data &&(response.status === 201 || response.status === 200)) {
+    return response;
+    }else{
+      console.log("response.data.errors:", response.data.errors);
+      // showErrorToast(response.data.errors[0])
+      showErrorToast("Unexpected error")
```
Comment (srishtigautam21) [inline src/Helpers/Services/actions.jsx:226]: remove commented code

**Review by srishtigautam21 — APPROVED** [review]
The code is well structured, clean and understandable
good work 
Good to merge

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Pull request merged**